src/snowflake/connector/auth/by_plugin.py

```diff
@@ -197,6 +186,7 @@ def handle_timeout(
         account: str,
         user: str,
         password: str,
+        delete_params: bool = True,
```

Comment:
I'm thinking of hiding it from the public apis, moving it into kwargs.
let's discuss about it